website/templates/project/view_file.mako

```diff
@@ -182,7 +182,7 @@
             docId: ${ sharejs_uuid | sjson, n },
             userId: ${ user['id'] | sjson, n },
             userName: ${ user['fullname'] | sjson, n },
-            userUrl: ${ '/' + user['id'] + '/' | sjson, n },
+            userUrl: ${ '/' + (user['id'] or '') + '/' | sjson, n },
```

Comment:
That would be less silly. I like that in a bugfix.

Even less silly still would be to never serve up user-centric fields when user is not logged in. But I'm not familiar enough with the page for that. Will push in a moment.